### ISSUE TITLE
Feature request: More options for/remapping hotkeys (e.g. full screen toggle)

### ISSUE THREAD

**Issue by RavenMacDaddy** [nba-emu/NanoBoyAdvance#252]
Established examples for full screen toggle being ALT-ENTER and F11, as discussed in #205.

Remapping would be a good addition, not just for toggling full screen, but for hotkeys overall as well, alongside in-game controls.

Separated of course for usability, e.g.:

Config > Input >

Controls
Hotkeys

**Comment by xZabuzax** [comment]
I agree with this, having hotkeys to configure the whole emulator (like Snes9X, VisualBoyAdvance-M, etc...) then it would be nice, this includes remapping the hotkeys for full-screen, save/load states, etc...

I hope this is implemented :)

**Comment by fleroviux** [comment]
> @fleroviux - I agree with this, setting F1 to load and Shift+F1to save is CRAZY AND DANGEROUS. If you dont press the Shift key strongly you WILL load a game you wanted to save which is/will be a disaster.

I don't see how it's more dangerous than any of the alternatives. What key binding do you think would be better? There are 10 save slots, so the options really are limited. Shift+Fn to load and Fn to save for example is not better at all - it makes it easier to accidentally overwrite a save state and more cumbersome to quickly load a save state.

In-game save functionality should be regularly used either way to avoid losing a lot of progress.
